extension-manifest-v3/src/store/options.js

```diff
@@ -76,21 +119,20 @@ const Options = {
       });
 
       // Send update message to another contexts (background page / panel / options)
-      try {
-        chrome.runtime.sendMessage({
-          action: UPDATE_OPTIONS_ACTION_NAME,
-          options,
-        });
-      } catch (e) {
-        console.error(
-          `Error while sending update options to other contexts: `,
-          e,
-        );
-      }
+      chrome.runtime
+        .sendMessage({ action: UPDATE_OPTIONS_ACTION_NAME })
+        .catch(() => {});
+
       return options;
     },
-    observe: (_, options) => {
-      observers.forEach((fn) => fn(options));
+    observe: (_, options, prevOptions) => {
+      observers.forEach((fn) => {
+        try {
+          fn(options, prevOptions);
```

Comment:
should this be awaited?

Reply:
It should not. Observers are not related to each other and should be called as soon as possible. 

Moreover, this PR does not change a logic here, just moves `try {} catch {}` from the wrapper into this method.

Reply:
I'll add it.